pkg/config/config.go

```diff
@@ -14,16 +14,16 @@ var (
 		field.WithRequired(true),
 		field.WithIsSecret(true),
 	)
-	EnterpriseTokenField = field.StringField(
-		"enterprise-token",
-		field.WithDisplayName("Enterprise Token"),
-		field.WithDescription("The Slack user oauth token used to connect to the Slack Enterprise Grid Admin API"),
+	BusinessPlusTokenField = field.StringField(
+		"business-plus-token",
+		field.WithDisplayName("Business Plus Token"),
+		field.WithDescription("The Slack user oauth token used to connect to the Slack Admin API (Business+ or Enterprise Grid)"),
 		field.WithIsSecret(true),
 	)
 	SSOEnabledField = field.BoolField(
 		"sso-enabled",
 		field.WithDisplayName("SSO Enabled"),
-		field.WithDescription("Flag indicating that the SSO has been configured for Enterprise Grid Organization. Enables usage of SCIM API"),
+		field.WithDescription("Deprecated: Enterprise Grid features moved to baton-slack-enterprise. This connector focuses on Business+ plans."),
```

Comment:
I think we should just remove the field, or at least use `WithExportTarget(ExportTargetNone)` so it doesn't show up in the UI